readme.txt

```diff
@@ -0,0 +1,1414 @@
+=== WP REST API (WP API) ===
+Contributors: rmccue, rachelbaker
+Tags: json, rest, api, rest-api
+Requires at least: 3.9
+Tested up to: 4.4
+Stable tag: 1.2.4
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+Access your site's data through an easy-to-use HTTP REST API (Version 1, deprecated).
```

Comment:
@danielbachhuber I was under the impression that V2 is production ready now. Either way, there does need to be some note in the readme that there is a V2. Right now, when you Google "WordPress REST API" the old plugin result shows up first, and it'd be easy to install and use it without knowing you aren't on the right one.

Re: putting readme.txt in git, it'd be nice for it to be in git so that others can help contribute to it.